README.md

```diff
@@ -1,2 +1,22 @@
-# workflow-generator
-Generate a GitHub Actions workflow for MATLAB
```

Comment:
Have you ever encountered a README about workflow generation? (Looking for existing examples if possible.)

Reply:
Thank you! This would be helpful.